src/main/java/it/pagopa/transactions/utils/NodoOperations.java

```diff
@@ -181,8 +176,31 @@ private Mono<String> nodoActivationForUnknownPaymentRequest(
                                 );
                             }
 
+                            final LocalDate dueDate = LocalDate.now().plusYears(100);
+                            final NodoTipoDatiPagamentoPA datiPagamentoPA = nodoAttivaRPTResponse
+                                    .getNodoAttivaRPTRisposta()
+                                    .getDatiPagamentoPA();
+                            final String paName = Optional.ofNullable(datiPagamentoPA.getEnteBeneficiario())
+                                    .map(CtEnteBeneficiario::getDenominazioneBeneficiario)
+                                    .orElse("");
+                            final String description = Optional.ofNullable(datiPagamentoPA.getCausaleVersamento())
+                                    .orElse("");
+
                             return StOutcome.OK.value().equals(nodoAttivaRPTRResponse.getEsito())
-                                    ? Mono.just("")
+                                    ? Mono.just(
+                                            new PaymentRequestInfo(
+                                                    rptId,
+                                                    rptId.getFiscalCode(),
+                                                    paName,
+                                                    description,
```

Comment:
Same thing here, should we make `paName` and `description` optional?

Reply:
I would make them optional since they are optional in received nodo response.

Reply:
See: pagopa/pagopa-ecommerce-commons#18